tooling/metamodel-generator/src/main/java/org/hibernate/processor/HibernateProcessor.java

```diff
@@ -364,6 +365,11 @@ private void processClasses(RoundEnvironment roundEnvironment) {
 		}
 
 		for ( Element element : roundEnvironment.getRootElements() ) {
+			if (isModule( element ) ) {
+				context.logMessage( Diagnostic.Kind.WARNING,
+									"Element '" + element + "' is module. Pass it." );
+				continue;
+			}
```

Comment:
I don't think we want to completely exclude modules from processing.

I think this solution is better: https://github.com/hibernate/hibernate-orm/pull/8802

Reply:
well ... ok. let's try your solution. the main thing is that it works :-)